server/models/Tier.js

```diff
@@ -165,13 +169,7 @@ export default function(Sequelize, DataTypes) {
         },
 
         amountStr() {
-          let str;
-          if (this.presets) {
-            const minimumAmount = Math.min(this.presets[0], this.amount);
-            str = `${formatCurrency(minimumAmount, this.currency)}+`;
-          } else {
-            str = formatCurrency(this.amount, this.currency);
-          }
+          let str = `${formatCurrency(this.minimumAmount, this.currency)}+`;
```

Comment:
What happens if it's `null`?

@Betree Generally, should we assume it's `100` cents in the current currency.

Reply:
I'm not sure, some collectives offer a free tier to get people into specific group. But we could put `100` as default in the frontend

Reply:
@Betree & @znarf Should I add this?

Reply:
We'll not implement that in this PR.